tests/next_tests/integration_tests/feature_tests/cases.py

```diff
@@ -47,11 +47,10 @@
 
 
 # mypy does not accept [IDim, ...] as a type
-IField: TypeAlias = common.Field[[IDim], np.int64]  # type: ignore [valid-type]
-IJKField: TypeAlias = common.Field[[IDim, JDim, KDim], np.int64]  # type: ignore [valid-type]
-IJKFloatField: TypeAlias = common.Field[[IDim, JDim, KDim], np.float64]  # type: ignore [valid-type]
-VField: TypeAlias = common.Field[[Vertex], np.int64]  # type: ignore [valid-type]
-EField: TypeAlias = common.Field[[Edge], np.int64]  # type: ignore [valid-type]
+IField: TypeAlias = common.Field[[IDim], np.float64]  # type: ignore [valid-type]
```

Comment:
I would keep an `int` type as indicated in the other PR, but not very strong opinion.